tools/cxo-observer/README.md

```diff
@@ -0,0 +1,77 @@
+# cxo-observer
```

Comment:
Can you add that for any and all support requests the affected resources from the output has to be attached